### ISSUE TITLE
Combining bind instructions - do not work

### ISSUE THREAD

**Issue by stanegit** [chrismichaelscott/iugo#9]
Hi, from document page model example
"
interests: {
        books: [
            {
                title: "20 thousand leagues under the sea",
                author: "Jules Verne",
                isbn: 92341
            }
        ]
"
and instructions

<div class="bindto-user" data-bind_key="address.city"></div>

<div class="bindto-interests">
    <ul data-bind_key="books">
        <li data-bind_each>
            <span data-bind_key="title"></span>
        </li>
    </ul>
</div>

dosn't work for me , it loops but I can not access titlte or author, looping works for simple arrays but subarray in objects do not

**Comment by stanegit** [comment]
Wow that was fast, thx I'll check it later

**Comment by chrismichaelscott** [comment]
Hi,

Thanks for picking up the issue. I believe there were two problems.

Firstly, the documentation had a typo in - a missing comma in the model. I've fixed that.

Assuming you had already spotted that there was a bug with nested tags that must have slipped in. Occurs if one uses `data-bind_each` attribute which is not directly under an element with `class="bindto-xxx"`. So,

```
<ul class="bindto-interests" data-bind_key="books">
```

would have worked.

Anyway, I think I've fixed the bug and I've added two tests which fail with the last release and pass now.

Let me know how it goes.
